projects/netgrif-components-core/src/lib/process/process.service.ts

```diff
@@ -27,6 +27,7 @@ export class ProcessService implements OnDestroy {
     protected _netUpdate: Subject<Net>;
     protected _requestCache: Map<string, ReplaySubject<Net>>;
     protected _referenceRequestCache: Map<string, ReplaySubject<PetriNetReferenceWithPermissions>>;
+    protected readonly LATEST = 'latest';
```

Comment:
the constant should be public, so that it can be referenced outside of the class if necessary